test-cases/longevity/longevity-lwt-500G-3d.yaml

```diff
@@ -1,5 +1,8 @@
 test_duration: 4450
-prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml n=400000000 ops'(insert_query=1)' cl=QUORUM -mode native cql3 -rate threads=1000" ]
+prepare_write_cmd: ["cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml n=400000000 ops'(insert_query=1)' cl=QUORUM -mode native cql3 -rate threads=100 -pop seq=1..133333333",
+                    "cassandra-stress user profile=/tmp/c-s_lwt_big_data.yaml n=400000000 ops'(insert_query=1)' cl=QUORUM -mode native cql3 -rate threads=100 -pop seq=133333333..266666666",
```

Comment:
n= is incorrect.
You need it to be 133333333.

Reply:
seq= should start from end+1.

Reply:
fixed